com.springrts.SpringLobby.yaml

```diff
@@ -0,0 +1,93 @@
+app-id: com.springrts.SpringLobby
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: springlobby
+rename-icon: springlobby
+rename-desktop-file: springlobby.desktop
+rename-appdata-file: springlobby.appdata.xml
+finish-args:
+- --share=network
+- --socket=x11
+- --share=ipc
+- --device=dri
+- --socket=pulseaudio
+- --filesystem=home
```

Comment:
What does it use home access for?

Reply:
It has to access `~/.spring/` for game files and `~/.springlobby/` its own settings and game scripts. If it is possible to restrict it, to those directory, this still should work.

Reply:
You can use `--persist=.spring` and `--persist=.springlobby` instead.

Reply:
Thanks, changed `--filesystem=home` to `--persist=.spring` and `--persist=.springlobby`